docs/reST/ref/display.rst

```diff
@@ -189,26 +195,12 @@ required).
 
    .. versionadded:: 2.0.0 ``vsync``
 
-   Basic example:
+   The ``hwnd`` argument is an integer contains the window id (or handle) of
+   a foreign window. By setting this parameter, pygame display can be embedded
```

Comment:
of course, keep the part you're adding documentation for new arguments and new API